lib/rules/jsx-sort-props.js

```diff
@@ -14,6 +14,19 @@ function isCallbackPropName(name) {
   return /^on[A-Z]/.test(name);
 }
 
+var RESEVED_PROPS_LIST = [
+  'children',
+  'dangerouslySetInnerHTML',
+  'htmlFor',
+  'key',
+  'ref',
+  'style'
+];
+
+function isReservedPropName(name, list) {
```

Comment:
I was playing with making it a "default" option if `list` wasn't specified; would you prefer that?

Regarding your general comment, I think it's important to allow the user to customize this list since some people may only want certain "reserved" props first. For example, maybe we want `key` first but all the rest to be alphabetized. I'm open to not allowing that (or, maybe better, to allowing overriding the list with only a subset of the reserved props), but I think without this option this  option won't be as useful.

```
function isReservedProp(name, list) {
  var listToUse = list || RESERVED_PROPS_LIST;
  return list.indexOf(name) >= 0;
}
```